astroquery/gemini/core.py

```diff
@@ -8,24 +8,19 @@
 
 from datetime import date
 
-from astropy import log
+from astroquery import log as logger
 from astropy import units
 from astropy.table import Table, MaskedColumn
 
 from astroquery.gemini.urlhelper import URLHelper
 import numpy as np
 
-import logging
-
 from ..query import BaseQuery, QueryWithLogin
 from ..utils.class_or_instance import class_or_instance
 from . import conf
 from ..exceptions import AuthenticationWarning
 
 
-logger = logging.getLogger(__name__)
```

Comment:
in `cacd` you added `'cadc'`, so maybe add 'gemini' here? 

I don't have a strong preference which approach to keep, but I suppose having consistency all over the modules would be best which would mean the removal of `logger` from cadc, as well as keeping it `log` here rather than `logger`?

Reply:
I moved the `gemini` logger to the main astroquery logger, but did not do that for the `cadc` one. I think I had a good reason for this, related to a test failure that I wasn't able to understand. I would rather ask the `cadc` developers to look into issue.

I will push a fix for the use of `from astroquery import log` instead of `from astroquery import log as logger`.